sdk/sdk-api/src/main/java/mil/emp3/api/KMLS.java

```diff
@@ -117,12 +117,13 @@ public KMLS(Context context, String serviceURL, IKMLSEventListener listener) thr
      * @throws MalformedURLException
      * @throws IllegalArgumentException
      */
-    public KMLS(Context context, String serviceURL, IKMLSEventListener listener, UUID geoId) throws MalformedURLException {
+    public KMLS(final Context context, final String serviceURL, final IKMLSEventListener listener, final UUID geoId) throws MalformedURLException {
         this(context, serviceURL, listener);
         if(null == geoId) {
             throw new IllegalArgumentException("geoId must be non-null");
         }
-        setGeoId(geoId);
+//        This call literally just throws an exception...
```

Comment:
Looks like this should just be removed. Otherwise make a not about why we're keeping it around.